rfcs/rfc-colin-states-mapping-to-lear.md

```diff
@@ -0,0 +1,127 @@
+- Start Date: 2021-12-03
+- Target Major Version: 
+- Reference Issues: [10098](https://github.com/bcgov/entity/issues/4797)
+- Entity Issue: [8285](https://github.com/bcgov/entity/issues/14691)
+- Implementation PR: (leave this empty)
+
+# Summary
+
+COLIN represents a business state as two states which are referred to as an operational state(`CORP_OP_STATE` table) and a corp state(`CORP_STATE`) table.  In many instances, these two states are used in conjunction to drive business logic and to determine allowable business state transitions between `ACTIVE` and `HISTORICAL` states for a given business.
+
+LEAR will not be adopting the COLIN approach of using the `CORP_STATE` table.  
+
+This RFC will provide design details as to how LEAR will achieve the same functionality as COLIN without the use of the `CORP_STATE` table.
+
+Note that the [Business States RFC](rfc-business-states.md) as well the current implementation of the Legal API was used as a starting point for the creation of this document.
+
+#### COLIN Database Structure for Business States
+![colin_states.png](rfc-colin-states-mapping-to-lear/colin_states.png)
+
+#### Current Corp State Data in CPRD
+![colin_corp_state_stats.png](rfc-colin-states-mapping-to-lear/colin_corp_state_stats.png)
```

Comment:
Could you clarify that the first column is the CORP_STATE and the second column is the CORP_OP_STATE?

Reply:
sure, will update

Reply:
updated.  first and second column are actually the reverse.